Source/Widgets/InfoBox/InfoBoxDescription.css

```diff
@@ -0,0 +1,42 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.cesium-infoBox-description {
+    font-family: sans-serif;
+    font-size: 13px;
+    padding: 4px 10px;
+    margin-right: 4px;
+    color: #edffff;
+}
+
+.cesium-infoBox-description table {
+    color: #edffff;
```

Comment:
@emackey I'll add the lighter class we talked about in the KML branch once this is in master.  I experimented a bit and I think it should work in most use cases.